crates/full-node/sov-sequencer/src/preferred/timestamp.rs

```diff
@@ -109,6 +109,8 @@ where
                 .try_into()
                 .expect("Converting unix timestamp to i64 number of milliseconds failed");
 
+            // Artificially inflate the generation number to reduce the number of tx hashes we need to store
+            let generation = timestamp.saturating_mul(100);
```

Comment:
Very minor nit: maybe we should do 


```suggestion
            let generation = timestamp.checked_mul(100).unwrap_or_else(|| { tracing::error!("Overflow multiplying timestamp by 100"); timestamp });
```
? If overflow ever happens it'd be nice to get explicit logs of this rather than slowing down to a crawl with every single transaction getting the same max generation number.

Not a practical concern though so non-blocking

Reply:
Going to leave this as is since it shouldn't happen for the next 3 million years or so :). Plus, we'll get a hard error anyway when the bucket fills up.